spec/lib/good_job/probe_server_spec.rb

```diff
@@ -11,12 +11,20 @@
   end
 
   describe '#start' do
-    it 'starts a webrick server' do
+    it 'starts a webrick server that binds to all interfaces' do
       probe_server.start
       wait_until(max: 1) { expect(probe_server).to be_running }
 
-      response = Net::HTTP.get("127.0.0.1", "/", port)
-      expect(response).to eq("OK")
+      ip_addresses = Socket.ip_address_list.select(&:ipv4?).map(&:ip_address)
```

Comment:
Not sure if this is too much magic 🧙 But I wanted to test this somehow...

Reply:
I like it! 👍